packages/webview_flutter/tizen/src/webview.cc

```diff
@@ -82,6 +177,14 @@ WebView::WebView(flutter::PluginRegistrar* registrar, int viewId,
     currentUrl_ = "about:blank";
   }
 
+  auto settings = params[flutter::EncodableValue("settings")];
+  if (std::holds_alternative<flutter::EncodableMap>(settings)) {
+    auto settingList = std::get<flutter::EncodableMap>(settings);
+    if (settingList.size() > 0) {
+      ApplySettings(settingList);
```

Comment:
ApplySettings can throw an error and This is in the constructor. Where should the error be handled?

Reply:
Why did you mention the constructor? I can't get your point

Reply:
https://stackoverflow.com/questions/5743013/exception-handling-a-constructor.Now I got your point.

Reply:
I fixed that it is handled in webview_factory.cc. Thanks